hack/deploy-sbo-local.sh

```diff
@@ -27,7 +27,7 @@ SBO_PID=$!
 attempts=12
 while [ -z "$(grep 'Starting workers' $SBO_LOCAL_LOG)" ]; do
     if [[ $attempts -ge 0 ]]; then
-        sleep 5
```

Comment:
@pmacik how about increasing attempts instead of increasing sleep time, that way we could save 5 secs if step is in done state.

Reply:
@akashshinde Sure, np... done